Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -28,10 +28,10 @@ class LibraryViewController: UIViewController {
     // UI Elements
     lazy var librarySegmentControl: UISegmentedControl = {
         var librarySegmentControl: UISegmentedControl
-        librarySegmentControl = UISegmentedControl(items: [UIImage(named: "library-bookmark")!,
-                                                           UIImage(named: "library-history")!,
-                                                           UIImage(named: "library-downloads")!,
-                                                           UIImage(named: "library-readinglist")!])
+        librarySegmentControl = UISegmentedControl(items: [UIImage(named: ImageIdentifiers.libraryBookmars)!,
+                                                           UIImage(named: ImageIdentifiers.libraryHistory)!,
+                                                           UIImage(named: ImageIdentifiers.libraryDownloads)!,
+                                                           UIImage(named: ImageIdentifiers.libraryReadingList)!])
```

Comment:
Do they have to be forced with `!` ? Maybe this could be an unwrapped computed var that returns an array of images?